dist/index.html

```diff
@@ -8,8 +8,27 @@
     <script src="../src/index.js"></script>
   </head>
   <body>
-    <div class="container">
-      <h1 class="center">Hello World!</h1>
+    <div class="parent-container">
+      <!-- <h1 class="center">Hello World!</h1> -->
+      <div class="navigation-container">
+        <button type="button" class="nav">Home</button>
+        <button type="button" class="nav">Projects</button>
+        <button type="button" class="nav">About Me</button>
+        <button type="button" class="nav">Contact</button>
+      </div>
+      <span class="perks">
+        Custom Designs · Marketing · Branding · And More
+      </span>
+      <div class="whois">
+        <h2> WHO IS SEEK-A-DESIGN?</h2>
+        </br>
```

Comment:
Let's try not to rely on `<br/>` tags to create horizontal spacing between elements. It's better if we begin building the habit of using CSS for this.

This is what's known as keeping a **"separation of concerns,"** and will become more important as our code grows. 

- We'll let our HTML be the nouns; "What is in our app?"
- We'll let our CSS be the adjective; "What are our nouns (the HTML) like?" 
- We'll let our Javascript be the verbs; "What do our nouns (the HTML) do?"